examples/msrollback.c

```diff
@@ -17,8 +18,8 @@ int main(void) {
 
     // msrollback can be used to remove, and destroy, multiple allocations from the memstack.
     // Below, we destroy "c" and "d", and remove "a" and "b".
-    msrollback(example_memstack, 2, TRUE);  // Destroy last two nodes
-    msrollback(example_memstack, 2, FALSE);  // Remove last two nodes
+    msrollback(example_memstack, 2, true);  // Destroy last two nodes
```

Comment:
These are stdbool booleans, which we try to avoid(I have a personal grudge against stdbool)

Reply:
why do you hate stdbools?